src/components/Epr.js

```diff
@@ -1,12 +1,17 @@
 import React from 'react';
 import { Event } from '../services/eventServices';
+import LocationForm from '../components/LocationForm';
 
 export default class Epr extends React.Component {
     constructor(props) {
         super(props);
 
         this.state = new Event();
         this.org = props.match.params.org;
+
+        this.addLocation = this.addLocation.bind(this);
```

Comment:
Not a change request, but might be interesting for you to know: using ES6 class fields, you don't need to bind all of your methods to preserve the context of `this`. Instead of something like `removeLocation(index){....}` you can do `removeLocation = (index) => {...}` and avoiding binding. pretty neat stuff! here's a stackoverflow post about it: https://stackoverflow.com/questions/32192682/react-js-es6-avoid-binding-this-to-every-method